sycl/CMakeLists.txt

```diff
@@ -157,16 +157,18 @@ option(SYCL_ENABLE_MAJOR_RELEASE_PREVIEW_LIB "Enable build of the SYCL major rel
 
 # Needed for feature_test.hpp
 if ("cuda" IN_LIST SYCL_ENABLE_PLUGINS)
-  set(SYCL_BUILD_PI_CUDA ON)
+  set(SYCL_BUILD_CUDA ON)
```

Comment:
Would it be better to add `adapter` here right away? Or maybe something like `backend`? Because it may be a bit confusing what does it mean "sycl build cuda".

But that's just a though, I will leave a final word to the code owners.

Reply:
I think using `backend` terminoligy in the SYCL build would make most sense, personally.